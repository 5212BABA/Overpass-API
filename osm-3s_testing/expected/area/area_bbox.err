encoding remark: Please enter your query and terminate it with CTRL+D.
runtime error: Filters too weak in query statement: specify in addition a bbox, a tag filter, or similar.
