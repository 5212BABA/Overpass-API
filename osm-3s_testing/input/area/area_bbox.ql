area(51.2,8.1,51.3,8.2);out geom;
