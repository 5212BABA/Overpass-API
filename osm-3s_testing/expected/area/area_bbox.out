<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="Overpass API">
<note>The data included in this document is from www.openstreetmap.org. The data is made available under ODbL.</note>
<meta osm_base=""/>

  <way id="602">
    <bounds minlat="51.2500000" minlon="8.1500000" maxlat="51.2500100" maxlon="8.1500200"/>
    <nd ref="60101" lat="51.2500100" lon="8.1500100"/>
    <nd ref="60102" lat="51.2500000" lon="8.1500000"/>
    <nd ref="60103" lat="51.2500000" lon="8.1500200"/>
    <nd ref="60101" lat="51.2500100" lon="8.1500100"/>
    <tag k="foo" v="bar"/>
  </way>

</osm>
